scripts/image/test-outbound-connectivity.sh

```diff
@@ -0,0 +1,163 @@
+# Create output directories up‑front so failures don’t leave us empty‑handed
+mkdir -p "$PCAP_DIR" "$PY_LOG_DIR"
+
+# ---------- parse flags (optional --cleanup) --------------------------
+CLEANUP=0
+if [[ "${1:-}" == "--cleanup" ]]; then
+  CLEANUP=1
+  shift
+fi
+
+SCENARIO="${1:-}"
+if [[ -z "$SCENARIO" ]]; then
+  echo "Usage: $0 [--cleanup] {baseline|missing-models|offline|offline-and-missing-models}" >&2
+  exit 1
+fi
+
+# ---------- optional pre‑run cleanup ----------------------------------
+if (( CLEANUP )); then
+  echo ">>> Removing leftover sut_* containers…"
+  docker rm -f $(docker ps -aq --filter name='^sut_') 2>/dev/null || true
+fi
+
+# ---------- scenario‑specific settings --------------------------------
+DO_NOT_TRACK=""; HF_HUB_OFFLINE=""; REMOVE_CACHE=0
+case "$SCENARIO" in
+  baseline) ;;
+  missing-models)               REMOVE_CACHE=1 ;;
+  offline)                      DO_NOT_TRACK=true; HF_HUB_OFFLINE=1 ;;
+  offline-and-missing-models)   DO_NOT_TRACK=true; HF_HUB_OFFLINE=1; REMOVE_CACHE=1 ;;
+  *) echo "Unknown scenario: $SCENARIO"; exit 1 ;;
+esac
+
+docker network inspect "$NET" >/dev/null 2>&1 || docker network create "$NET"
+
+# ---------- launch SUT idle -------------------------------------------
+CID=$(docker run -d --rm --name "sut_${SCENARIO}" \
+         --network "$NET" \
+         --cap-add NET_RAW --cap-add NET_ADMIN \
+         -e DO_NOT_TRACK="$DO_NOT_TRACK" \
+         -e HF_HUB_OFFLINE="$HF_HUB_OFFLINE" \
+         --entrypoint /bin/sh "$IMAGE" -c "sleep infinity")
+echo "Container: $CID  (scenario $SCENARIO)"
+
+# install tcpdump (Wolfi uses apk) as root
+docker exec -u root "$CID" apk add --no-cache tcpdump >/dev/null
+
+# optionally wipe HF cache
+(( REMOVE_CACHE )) && docker exec "$CID" rm -rf "$HF_CACHE" || true
+
+# ---------- start tcpdump in background -------------------------------
+FILTER='not (dst net ff02::/16 or src net ff02::/16 or ip6[6] = 58 or ether multicast)'
+
+docker exec -u root -d "$CID" sh -c "tcpdump -U -n -i $CAPTURE_IFACE '$FILTER' -w /tmp/capture.pcap > /tmp/tcpdump.log 2>&1"
+
+# check if tcpdump stayed alive
+sleep 2
+if ! docker exec "$CID" pgrep tcpdump >/dev/null; then
+  echo 'tcpdump exited – showing its log:'
+  docker exec "$CID" cat /tmp/tcpdump.log
+  exit 1
+fi
+
+echo "tcpdump running on interface $CAPTURE_IFACE..."
+# ---------- run the Python workload -----------------------------------
+echo ">>> Running Python workload (capturing stdout/stderr)…"
+#   ‑ The "|&" pipes *both* stdout *and* stderr into tee.
+#   ‑ tee sends it to the terminal *and* writes the log file.
+#   ‑ With `set -o pipefail` we still fail early if the Python process exits non‑zero.
+
+if [[ "$HF_HUB_OFFLINE" -eq 1 && "$REMOVE_CACHE" -eq 1 ]]; then
+  echo "HF_HUB_OFFLINE=1 and REMOVE_CACHE=1 : allowing python command have a non-exit 0 status and will continue the script."
+  set +e
+fi
+
+
+docker exec -i -e PYTHONUNBUFFERED=1 "$CID" python - <<PY |& tee "${PY_LOG_DIR}/${SCENARIO}.log"
```

Comment:
I have the version of bash that came with my Mac, and |& isn't a valid symbol. If there's an equivalent construction that works with the old (<4) version of bash, it might be worth using it for compatibility.